### DISCUSSION TITLE
Local deployment error:ValueError: too many values to unpack (expected 2)

### DISCUSSION THREAD

**Discussion by SaltedCarl** [zai-org/ChatGLM3#1288] [Q&A]
I've tried to deploy chatglm3-6b for several times, sometimes it works well, but sometimes it crashes like this. it crashes at the quickstart.
Thanks for help!
![Screenshot2](https://github.com/THUDM/ChatGLM3/assets/140713707/49908ea3-12c1-479d-9219-32a5fe67283d)
![Screenshot1](https://github.com/THUDM/ChatGLM3/assets/140713707/83568d79-f862-4824-b0d0-590e6e94ec00)

full crash reort attached here:
[Crash_Report.txt](https://github.com/user-attachments/files/16173332/Crash_Report.txt)

**Comment by ts-haiming-zhang** [comment]
It works well after changed as following
torch                         2.1.2+cu121
transformers                  4.38.2
